.github/workflows/nodejs-ci.yml

```diff
@@ -85,6 +91,16 @@ on:
         description: "Command to execute at the end of the container scan"
         type: string
         default: ""
+      send_test_coverage_report_to_pulse:
```

Comment:
Can we rename it to `send_dev_test_coverage_report_to_pulse` to keep in sync with other dev_test vars?

Reply:
YEs, did the changes